proto/warg/transparency/proofs.proto

```diff
@@ -24,4 +24,14 @@ message LogProofBundle {
 message HashEntry {
     uint32 index = 1;
     bytes hash = 2;
-}
\ No newline at end of file
+}
+
+message StackLog {
```

Comment:
Are we even using StackLog any more? I don't see any references to it outside of `warg-transparency`...

Reply:
Yes, we use the `StackLog` for computing the `logRoot` on Checkpoints. `VecLog` is used for computing the log proof bundles.

Reply:
Is that happening somewhere in this repo? I just don't see any references to `StackLog` outside of this crate...

Reply:
It looks like *I* removed it in https://github.com/bytecodealliance/registry/pull/169 though I have no recollection of that :sweat_smile:

Reply:
Yes, looks like you may be correct. Looks like the `CoreService` uses the `VecLog` for computing the `logRoot`. Both `StackLog` can compute the `logRoot`. From my understanding, the `StackLog` memory usage scales differently than the `VecLog`.

There's an argument to be made for dropping unused code. I'm currently using both in my server implementation. It would be helpful to keep both around and revisit this.

Reply:
OK - part of the reason I was asking is that it feels wrong to expose otherwise-hidden implementation details here where we are otherwise defining a public interface to the service. It might be worthwhile to put these messages in a separate file (`log_impls.proto`?) and hide the generated code behind a feature flag. If that is too big a lift right now I think comments on the message types here to the effect of "these are internal implementation details of the server, not part of the client contract" would be OK in the medium-term.

Reply:
Understand. I'll look into it.

Reply:
Pushed an update. Created an `proto/warg/internal/internal.proto`. That probably is enough and doesn't need to be behind a feature flag, but open to doing that. Let me know.